docs/contributing/docs-writing-process.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Docs Writing Process
+---
+
+When a new feature or integration is released that Gatsby developers can take advantage of, documentation should be added to improve the learning experience. Gatsby also needs docs for topic areas that have existed for periods of time in other technologies but are under-documented or under-tooled in the Gatsby ecosystem.
+
+The knowledge of how to work with a technique, source plugin, or varied use case may be known internally to Gatsby team members, but it also may only exist on the web at large. This contributing doc is intended to provide a written process for producing docs without prior information, a critical function of the Gatsby Learning team and open source community.
+
+## Identifying a topic
+
+When identifying a topic, start by:
+
+1. Looking at GitHub issues. Good labels to seek out are [`help wanted`](https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+) and [`good first issue`](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+   - This part may be covered if a docs issue is assigned to you, or if you’re signing up voluntarily to take on an issue.
+
+2. Look at the [stub list](/contributing/stub-list/) to find docs that need contributions.
+
+3. Look at the [learning workflow meta issue](https://github.com/gatsbyjs/gatsby/issues/13708) to find active areas looking for docs.
+
+4. Read through the existing Gatsby docs information and find gaps in topic coverage. Is there an area you feel is missing? [File an issue](/contributing/how-to-file-an-issue/) first to discuss, then implement in a PR.
+
+5. Observe common points of confusion or rough edges through user feedback and recommend solutions.
+
+> _Note: It’s required to open a GitHub issue before submitting a PR if one does not already exist._
+
+## Selecting the correct format
+
+A GitHub issue for new learning material should indicate the format. Is it a Reference or Conceptual Guide? A Tutorial? A recipe?
```

Comment:
I could add some links. But ever since your comment about branching while reading, I've limited how many links I put inline. So they'd likely come after the section.

Reply:
Didn't realize I had such an effect! In my mind there is actually a bit of a difference here. I consider this glossary type links. If people need more context, they'll click. Versus links we expect them to click on and review in addition to what they're currently reading.